src/components/Todo.js

```diff
@@ -11,18 +11,20 @@ class Todo extends React.Component {
             title,
             id
          } = this.props;
-         
-        return (
+
+         return (
             <div>
-                <li className={`todo-wrapper`}>
-                    {title}
-                    <input
-                        className={`todo-button`}
-                        type='submit'
-                        id={id}
-                        value={valueButton}
-                        onClick={todoRemove} />
-                </li>
+                {!!title &&
```

Comment:
Why is `title` empty? Could that be filtered before hands or is such data returned at all?